src/metpy/calc/cross_sections.py

```diff
@@ -292,12 +292,14 @@ def absolute_momentum(u, v, index='index'):
 
     """
     # Get the normal component of the wind
-    norm_wind = normal_component(u, v, index=index).metpy.convert_units('m/s')
+    u = u.metpy.convert_units('m/s')
+    v = v.metpy.convert_units('m/s')
```

Comment:
Maybe change these to e.g. `u.metpy.quantity()`? Then the last line of the function could be:

```python
return (norm_wind + f * distance).metpy.convert_units('m/s')
```